.github/workflows/main.yml

```diff
@@ -0,0 +1,67 @@
+name: update-translations
+
+on:
+  push:
+    branches:
+      - master
+
+# on:
+#   schedule:
+#     - cron: "0 0 * * *" #runs at 00:00 UTC everyday
```

Comment:
I implemented it to run with each master change to make sure it works, after that, I will change it to time-base.

Reply:
done ! https://github.com/sphinx-doc/sphinx-doc-translations/commit/63d223c0aeb8dcba361f617ff031f3a87a142894